### ISSUE TITLE
Allow saving directly as a file

### ISSUE THREAD

**Issue by julianfairfax** [you-apps/ConnectYou#266]
### Describe your suggested feature

Currently, the export dialogue presents the Android share sheet, which doesn't include an option to save the contact as a file directly. It would be nice if Connect You provided this functionality.

My personal use case is exporting individual contacts to import them into Proton Contacts (which currently has no sync functionality for Android).

It seems that Proton Contacts will create duplicates when importing a previous Proton Contacts export, but will when importing a Connect You export (even when having previously imported that same export into Proton Contacts - I believe this is to due to Proton Contacts giving each contact a UID, something Connect You doesn't preserve on import from Proton Contacts, and thus doesn't export).

Connect You, on the contrary, is both able to successfully import previous Connect You exports without creating duplicates, and even import Proton Contacts imports without any duplicates!

In conclusion, it would be nice to be able to export an individual contact from Connect You for importing into Proton Calendar, since the resulting merge operation (or lack thereof is this is a new contact) is easier and faster with one contact, than with all contacts.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.
